File: SpecialEducationPlanning/SpecialEducationPlanning.Web/src/app/shared/models/Education-specification.model.ts
import { HousingSpecificationTemplates } from "./housing-specification-templates.model";
import { HousingType } from "./housing-type.model";
import { Project } from "./project";

export interface HousingSpecification {
    projectId: number;
    code?: string;
    name?: string;
    planState: number;
    projectModel: Project;
    housingTypes: HousingType[];
    housingSpecificationTemplates: HousingSpecificationTemplates[];
    id: number;
}

File: SpecialEducationPlanning/SpecialEducationPlanning.Business/Model/Project/EducationSpecificationModel.cs
using Koa.Domain;
using System.Collections.Generic;

namespace SpecialEducationPlanning
.Business.Model;

public class HousingSpecificationModel : BaseModel<int>
{
    public int ProjectId { get; set; }
    public string Code { get; set; }
    public string Name { get; set; }
    public int PlanState { get; set; }
    public virtual ProjectModel ProjectModel { get; set; }
    public virtual ICollection<HousingTypeModel> HousingTypes { get; set; }
    public virtual ICollection<HousingSpecificationTemplatesModel> HousingSpecificationTemplates { get; set; }
}
